.github/workflows/master.yml

```
@@ -0,0 +1,32 @@
+name: master
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up java 15
+        uses: actions/setup-java@v1
+        with:
+          java-version: 15
+      - name: build
+        run: mvn -B verify --file pom.xml
+  publish:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up java 15
+        uses: actions/setup-java@v1
+        with:
+          java-version: 15
+      - name: build
+        run: >
+          mvn -B verify jib:build
```

```suggestion
          mvn -B compile jib:build
```
Workflow already makes a verification on the previos step. No need to make it again.